resources/php_agent.rb

```diff
@@ -28,7 +28,6 @@
 
 attribute :mode,           kind_of: [Integer, String], default: 0o644
 
-attribute :cookbook,       kind_of: String, default: 'newrelic-ng'
```

Comment:
This removes the ability of providing a custom source from the caller's cookbook. I'm not sure if this is used by anyone, but at least in theory this provides a bit more flexibility. I'm not using `php_agent` myself, so if you say that it makes no sense to have a custom `newrelic.ini.php`, then I'm willing to drop this.